src/resdk/tables/rna.py

```diff
@@ -21,10 +21,196 @@
+        "slug": "avg_seq_length_raw",
+        "type": "float64",
+        "agg_func": "mean",
+    },
+    {
+        "name": "FastQC (trimmed)_mqc-generalstats-fastqc_trimmed-avg_sequence_length",
+        "slug": "avg_seq_length_trimmed",
+        "type": "float64",
+        "agg_func": "mean",
+    },
+    {
+        "name": "STAR_mqc-generalstats-star-uniquely_mapped_percent",
+        "slug": "mapped_reads_percent",
+        "type": "float64",
+        "agg_func": "mean",
+    },
+    {
+        "name": "STAR_mqc-generalstats-star-uniquely_mapped",
+        "slug": "mapped_reads",
+        "type": "Int64",
+        "agg_func": "sum",
+    },
+    {
+        "name": "STAR (Globin)_mqc-generalstats-star_globin-uniquely_mapped_percent",
+        "slug": "mapped_reads_percent_globin",
+        "type": "float64",
+        "agg_func": "mean",
+    },
+    {
+        "name": "STAR (Globin)_mqc-generalstats-star_globin-uniquely_mapped",
+        "slug": "mapped_reads_globin",
+        "type": "Int64",
+        "agg_func": "sum",
+    },
+    {
+        "name": "STAR (rRNA)_mqc-generalstats-star_rrna-uniquely_mapped_percent",
+        "slug": "mapped_reads_percent_rRNA",
+        "type": "float64",
+        "agg_func": "mean",
+    },
+    {
+        "name": "STAR (rRNA)_mqc-generalstats-star_rrna-uniquely_mapped",
+        "slug": "mapped_reads_rRNA",
+        "type": "Int64",
+        "agg_func": "sum",
+    },
+    {
+        "name": "featureCounts_mqc-generalstats-featurecounts-percent_assigned",
+        "slug": "fc_assigned_reads_percent",
+        "type": "float64",
+        "agg_func": "mean",
+    },
+    {
+        "name": "featureCounts_mqc-generalstats-featurecounts-Assigned",
+        "slug": "fc_assigned_reads",
+        "type": "Int64",
+        "agg_func": "sum",
+    },
+    {
+        "name": "Salmon_mqc-generalstats-salmon-percent_mapped",
+        "slug": "salmon_assigned_reads_percent",
+        "type": "float64",
+        "agg_func": "mean",
+    },
+    {
+        "name": "Salmon_mqc-generalstats-salmon-num_mapped",
+        "slug": "salmon_assigned_reads",
+        "type": "Int64",
+        "agg_func": "sum",
+    },
+    {
+        "name": "QoRTs_mqc-generalstats-qorts-Genes_PercentWithNonzeroCounts",
+        "slug": "nonzero_count_features_percent",
+        "type": "float64",
+        "agg_func": "mean",
+    },
+    {
+        "name": "QoRTs_mqc-generalstats-qorts-NumberOfChromosomesCovered",
+        "slug": "contigs_covered",
+        "type": "Int64",
+        "agg_func": "mean",
+    },
+    {
+        "slug": "strandedness_code",
+        "type": "string",
+    },
+    {
+        "slug": "genome_build",
+        "type": "string",
+    },
+]
+
+
+def multiqc_general_stats_parser(file_object, name):
+    """Parse "multiqc_general_stats.txt" file."""
+    df = pd.read_csv(file_object, sep="\t", index_col=0)
+
+    # Keep only specified columns:
+    df = df[
+        [
+            c.get("name", "")
```

Comment:
I know it is tempting to use short variable names, but I prefer more descriptive ones.
Such as `data_frame` instead of `df`, `column` instead of `c`, `aggregate??` instead of `s` ...

I believe it makes the code more readable.

Reply:
I've done this for all but `df`, which is IMO the community accepted notation for a `pandas.DataFrame`.